### PR TITLE
Test: Set up Jest and add contact sales form tests

### PR DESCRIPTION
**Pull request by natapg**
Closes [WEB-318](https://fpjs.myjetbrains.com/youtrack/issue/WEB-318)

### CODE REVIEW

**Comment by Lisenish** [comment]
I noticed this:
image

Why do we use backtick character there? I believe it should be just regular single quote, like "We'll" not "We`ll"

**Review by Lisenish — APPROVED** [review]

**Comment by Lisenish** [comment]
@natapg CI check doesn't work?
image

File: package.json [hunk 1]
```diff
@@ -110,7 +117,11 @@
     "eslint-webpack-plugin": "3.1.1",
     "gatsby-plugin-eslint": "4.0.2",
     "gatsby-plugin-svg-sprite": "2.0.1",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^27.4.7",
+    "jest-fetch-mock": "3.0.3",
     "prettier": "2.5.1",
+    "react-test-renderer": "^17.0.2",
```
[the remark this answers is not in the document]
Comment (natapg) [reply package.json:124]: you are right, It's a leftover from Gatsby's guide, I'll remove it

**Comment by natapg** [comment]
> @natapg CI check doesn't work? image

I don't know @Lisenish , the tests passed in the action, let me check

**Pull request merged**